code/game/objects/effects/spawners/structure.dm

```diff
@@ -163,6 +163,11 @@ again.
 	icon_state = "plastitaniumwindow_spawner"
 	spawn_list = list(/obj/structure/grille, /obj/structure/window/plastitanium)
 
+//plastitanium pirate window
+
+/obj/effect/spawner/structure/window/plastitanium/pirate
+	explosion_block = 50
```

Comment:
Remove this line, it's a spawner and its explosion_block value is totally meaningless.